config/example/game_options.txt

```diff
@@ -38,8 +38,8 @@ REVIVAL_BRAIN_LIFE -1
 ## We suggest editing these variabled in-game to find a good speed for your server. To do this you must be a high level admin. Open the 'debug' tab ingame. Select "Debug Controller" and then, in the popup, select "Configuration". These variables should have the same name.
 ## These values get directly added to values and totals in-game. To speed things up make the number negative, to slow things down, make the number positive.
 
-## These modify the run/walk speed of all mobs before the mob-specific modifiers are applied. 
-RUN_SPEED 1
+## These modify the run/walk speed of all mobs before the mob-specific modifiers are applied.
+RUN_SPEED 1.7
 WALK_SPEED 4
```

Comment:
This works, though. Changing human_delay to 1.7 would end with a base move speed of 2.7 for humans, since they apply living/movement_delay() now.